src/config/kernel.md

```diff
@@ -26,9 +26,9 @@ $ vkpurge list
 3.8.2_1
 ```
 
-You can now remove a specific kernel version like `3.8.2_1` or `all` which
-removes all kernels expect the latest kernel of each series and the kernel that
-is currently booted.
+To remove a specific kernel version like `3.8.2_1` or `all` (which removes all
```

Comment:
This is not exactly a sentence. Maybe something like:

To remove a specific version:
```
vkpurge rm 3.8.2_1
```

To remove all kernels except the latest kernel of each series and the kernel that is curretly booted:
```
# vkpurge rm all
```

Reply:
And some text above `vkpurge list` would be helpful.